gapis/api/vulkan/state_rebuilder.go

```diff
@@ -2667,22 +2708,7 @@ func (sb *stateBuilder) createGraphicsPipeline(gp GraphicsPipelineObjectʳ) {
 				)).Ptr()),
 			tessellationState, // pTessellationState
 			viewportState,     // pViewportState
-			NewVkPipelineRasterizationStateCreateInfoᶜᵖ(sb.MustAllocReadData( // pRasterizationState
-				NewVkPipelineRasterizationStateCreateInfo(
-					VkStructureType_VK_STRUCTURE_TYPE_PIPELINE_RASTERIZATION_STATE_CREATE_INFO, // sType
-					0, // pNext
-					0, // flags
-					gp.RasterizationState().DepthClampEnable(),        // depthClampEnable
-					gp.RasterizationState().RasterizerDiscardEnable(), // rasterizerDiscardEnable
-					gp.RasterizationState().PolygonMode(),             // polygonMode
-					gp.RasterizationState().CullMode(),                // cullMode
-					gp.RasterizationState().FrontFace(),               // frontFace
-					gp.RasterizationState().DepthBiasEnable(),         // depthBiasEnable
-					gp.RasterizationState().DepthBiasConstantFactor(), // depthBiasConstantFactor
-					gp.RasterizationState().DepthBiasClamp(),          // depthBiasClamp
-					gp.RasterizationState().DepthBiasSlopeFactor(),    // depthBiasSlopeFactor
-					gp.RasterizationState().LineWidth(),               // lineWidth
-				)).Ptr()),
+			pipelineRasterizationState,
```

Comment:
`// pRasterizationState` - seems obvious, but we have it for all the other thigns.